definitions/infra-vspherevm/golden_metrics.yml

```diff
@@ -19,3 +19,24 @@ diskUsageMib:
     from: VSphereVmSample
     facet: entityName
     eventId: entityGuid
+overallStatus:
+  title: Overall Status of the VM
+  query:
+    eventId: entityGuid
+    select: latest(`overallStatus`)
```

Comment:
same question as the other overallStatus

Reply:
You're right, it's a string containing the status. useful for customer but not strictly a metric. should i remove it?